CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # bedrock-tokenization ChangeLog
 
+## 8.3.0 - 2022-TBD
```

Comment:
This needs to be a breaking change because it requires a breaking-change bedrock-tokenizer peer dependency.

Reply:
Changed to be a breaking change 5d51574